staging/src/k8s.io/client-go/tools/cache/controller.go

```diff
@@ -72,6 +72,9 @@ type Config struct {
 
 	// Called whenever the ListAndWatch drops the connection with an error.
 	WatchErrorHandler WatchErrorHandler
+	
+	// WatchListPageSize is the requested chunk size of initial and resync watch lists.
```

Comment:
We use "resync" terminology in the context of resyncing from in-memory state. What you means is relist.

Reply:
I copied this line from reflector.go :D Fixed here. I can fix in reflector.go as well if you want.